### PR TITLE
Var organization

### PR DESCRIPTION
**Pull request by BJSalvatore**
Just moved the eventListener to the top; the variables all follow that; then arrays; followed by the functions.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -353,7 +367,7 @@ function car_collision() {
 
 function water_collision() {
     for (var i = 15; i < 30; i = i+3) {
-        var obs = obstacleArray[i];
+        var obs = obstacleArray[i];- obs.dw/2
```
Comment (TheErikWolfe) [inline main.js]: I think this might be a typo. Please confirm. Should it have been obstacleArray[i] - obs.dw/2; ?

**Review by dimitri16h — COMMENTED** [review]
What is happening on lines 370 and 398? It looks like a part was taken out of if statement but then added after a semicolon?

File: main.js (hunk 1 above)
Comment (dimitri16h) [reply main.js]: Yeah thats what I checked as well, it appears to be moved from 398

Comment (BJSalvatore) [reply main.js]: Fixed that typo and pushed back to var-organization branch

**Review by dimitri16h — APPROVED** [review]

**Review by TheErikWolfe — APPROVED** [review]

**Review by kevin-moore-729 — APPROVED** [review]

**Pull request merged**